axis_linear_x: 1
axis_linear_y: 0
axis_angular_z: 2
scale_angular_z: 0.7
scale_linear_x: 0.7
scale_linear_y: 0.7
